src/tests/ffi.cpp

```diff
@@ -737,13 +737,13 @@ getpasscb_once(rnp_ffi_t        ffi,
 
 static bool
 getpasscb_inc(rnp_ffi_t        ffi,
-               void *           app_ctx,
-               rnp_key_handle_t key,
-               const char *     pgp_context,
-               char *           buf,
-               size_t           buf_len)
+              void *           app_ctx,
```

Comment:
@rrrooommmaaa  Please rebase on master so these changes which are already merged will not appear in PR.

Reply:
I applied clang-format on this file. (It wasn't applied on `master`, looks like)

Reply:
@rrrooommmaaa It was because this PR was started before the clang-format rule was added and run on the whole master. Anyway, now it looks good and almost ready to merge.